app/src/main/java/de/danoeh/antennapod/dialog/SleepTimerDialog.java

```diff
@@ -144,6 +170,25 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         return builder.create();
     }
 
+    private void updateAutoEnableText() {
```

Comment:
I think `updateAutoEnableText` would make it more clear what the method does

Reply:
That's how the method called :)

Reply:
Oops, sorry. I reviewed it from my phone and I guess it was somehow out of screen